lib/network/http2_client.go

```diff
@@ -122,6 +123,32 @@ func (c *HTTP2NetworkClient) SendMessage(message common.Serializable) (retBody [
 	return
 }
 
+func (c *HTTP2NetworkClient) SendTransaction(message common.Serializable) (retBody []byte, err error) {
+	headers := c.DefaultHeaders()
+	headers.Set("Content-Type", "application/json")
+
+	var body []byte
+	if body, err = message.Serialize(); err != nil {
+		return
+	}
+
+	u := c.resolvePath(resource.URLTransactions)
```

Comment:
`SendMessage` and `SendTransaction` are almost same except this part.
And `SendBallot` also.
I figure, if this duplication of code resolves, the code will get better.

Reply:
@Charleslee522 I think so. `SendMessage` is currently using for `Transaction` and 'Connect', so I made new.

Reply:
@spikeekips 
I mean,
```
func (c *HTTP2NetworkClient) SendTo(message common.Serializable, uri string) (retBody []byte, err error) {
...
}

func (c *HTTP2NetworkClient) SendMessage(message common.Serializable) (retBody []byte, err error) {
    return SendTo(message, UrlPathPrefixNode + "/message")
}

func (c *HTTP2NetworkClient) SendTransaction(message common.Serializable) (retBody []byte, err error) {
    return SendTo(message, resource.URLTransactions)
}

func (c *HTTP2NetworkClient) SendBallot(message common.Serializable) (retBody []byte, err error) {
    return SendTo(message, UrlPathPrefixNode + "/ballot")
}
```